src/components/organisms/AdminVenueView/components/EventsView/CsvImportModal.tsx

```diff
@@ -0,0 +1,219 @@
+            <span className="CsvImportModal__error-static">
+              There has been an error parsing the CSV file. Please make sure
+              everything is OK before proceeding. Error:
+            </span>
+            <br />
+            <span className="CsvImportModal__error-generated">
+              {parsedErrors[0].message}
+            </span>
+          </div>
+        )}
+        {hasParseError && parsedErrors && parsedErrors?.length > 1 && (
+          <div className="CsvImportModal__error CsvImportModal__error--multiple">
+            <span className="CsvImportModal__error-static">
+              There have been multiple errors parsing the CSV file. Please make
+              sure everything is OK before proceeding. Errors:
+            </span>
+            <br />
+            <span className="CsvImportModal__error-generated">
+              {parsedErrors.map((e) => e.message).join(". ")}
+            </span>
+          </div>
+        )}
+        <div className="CsvImportModal__wrapper">
+          {!parsedData && (
+            <span className="CsvImportModal__description">
+              The result of parsing the CSV file will be displayed here. You can
+              download an example to get you started
+            </span>
+          )}
+          {parsedData && (
+            <table className="CsvImportModal__table">
+              <tr>
+                <th>#</th>
+                {parsedMeta?.fields?.map((heading) => (
+                  <th key={heading}>{heading}</th>
+                ))}
+              </tr>
+              {parsedData.map((row, rowIndex) => (
+                <tr key={rowIndex}>
+                  <td>{rowIndex + 1}</td>
+                  {Object.values(row).map((value, cellIndex) => (
+                    <td key={cellIndex}>{value}</td>
+                  ))}
+                </tr>
+              ))}
+            </table>
+          )}
+        </div>
+      </Modal.Body>
+      <Modal.Footer className="CsvImportModal__footer">
+        <ButtonNG
+          className="CsvImportModal__example CsvImportModal__button"
+          onClick={downloadExample}
+        >
+          Download example CSV
+        </ButtonNG>
+        <FileButton
+          className="CsvImportModal__select CsvImportModal__button"
+          onClick={selectFile}
+          loading={isParsing}
+          disabled={isParsing}
+        >
+          Select CSV file...
+        </FileButton>
+        <ButtonNG
+          className="CsvImportModal__clear CsvImportModal__button"
+          onClick={clearParsed}
+          disabled={isParsing}
+        >
+          Clear
+        </ButtonNG>
+        <ButtonNG
+          className="CsvImportModal__import CsvImportModal__button"
+          title="Upload the selected events"
+          type="submit"
+          onClick={importEvents}
+          variant="primary"
+          loading={isImporting}
+          disabled={isParsing || isImporting || !parsedData?.length}
```

Comment:
Simplify it by extracting it into a single `const`

Reply:
using single constant for disabled @ 6b615b1a3a675579161eb2b4fe2f1f8b6e9daf47